main/src/main/java/org/mobilitydata/gtfsvalidator/notice/RouteSameNameAndDescriptionNotice.java

```diff
@@ -29,8 +29,8 @@
  *
  * <p>Severity: {@code SeverityLevel.ERROR}
  */
-public class SameNameAndDescriptionForRouteNotice extends ValidationNotice {
-  public SameNameAndDescriptionForRouteNotice(
+public class RouteSameNameAndDescriptionNotice extends ValidationNotice {
```

Comment:
I feel that RouteWithSameNameAndDescriptionNotice sounds more natural.

Reply:
Indeed! Will change this. thanks :)